src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/enable-containerized-function-monitoring/enable-containerized-function.mdx

```diff
@@ -0,0 +1,241 @@
+
+First, you need to create image layers. Once those layers are created you can either create a regular or a public ECR function. 
+
+<Steps>
+<Step>
+### Create a New Relic Lambda layer as an image [#create-lambda-layer]
+
+Before you can deploy your image, you first need to create a New Relic Lambda layer. To do this, complete the following steps: 
+
+1. Determine which Amazon Resource Name (ARN) layers are available [here](https://layers.newrelic-external.com/#/get-layers/get).
+2. Create a new docker image using the following code:
+```
+FROM alpine:latest as layer-copy 
+# Create a build stage to copy the files from S3 using credentials ARG AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION:-"region"} 
+ARG AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID:-"accessKey"} 
+ARG AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY:-"secretkey"} ENV AWS_DEFAULT_REGION=${AWS_DEFAULT_REGION} 
+ENV AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID} 
+ENV AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY} 
+RUN apk add aws-cli curl unzip 
+RUN mkdir -p /opt 
+RUN curl $(aws lambda get-layer-version-by-arn --arn <ARN Link> --query 'Content.Location' --output text) --output layer.zip 
+RUN unzip layer.zip -d /opt 
+RUN rm layer.zip 
+FROM scratch 
+# Start second stage from blank image to squash all previous history, including credentials. 
+WORKDIR /opt 
+COPY --from=layer-copy /opt .
+```
+3. Push that image to Amazon ECR. 
+
+```
+docker build -t layer-image1:latest .
+```
+</Step>
+
+<Step>
+### Deploy the image in AWS ECR [#deploy-image-ECR]
+
+Now that you've created the image, you can deploy it using the following steps: 
+
+1. Configure your AWS account from the terminal.
+2. Authenticate your account using the following command:
+
+```
+aws ecr get-login-password --region us-east-1 | docker login --username AWS --password-stdin 111122223333.dkr.ecr.us-east-1.amazonaws.com
+```
+3. Create a repository in Amazon ECR using the `create-repository` command:
+
+```
+aws ecr create-repository --repository-name hello-world --image-scanning-configuration scanOnPush=true --image-tag-mutability MUTABLE
+```
+4. Copy the `repositoryUri` from the output in the previous step.
+5. To tag your local image into your Amazon ECR repository as the latest version, run the docker tag command: 
+```
+Replace docker-image:test with the name and tag of your Docker image.
+```
+6. Replace &lt; ECRrepositoryUri &gt;  with the `repositoryUri` that you copied. Make sure to include `:latest` at the end of the URI. Here is an example: 
+
+```
+docker tag docker-image:test <ECRrepositoryUri>:latest
+```
+7. Run the docker push command to deploy your local image to the Amazon ECR repository. Make sure to include `:latest` at the end of the repository URI. 
+```
+docker push 111122223333.dkr.ecr.us-east-1.amazonaws.com/hello-world:latest
+```
+</Step>
+</Steps>
+
+## Create a Lambda function 
```

Comment:
It should be something like : Create a Containerized Lambda function with Newrelic layer image ECR